doc/geany.txt

```diff
@@ -2636,6 +2636,16 @@ reload_clean_doc_on_file_change   Whether to automatically reload documents    f
                                   on disk.
                                   If unsaved changes exist then the user is
                                   prompted to reload manually.
+save_config_on_file_change        Automatically save Geany's configuration     true        immediately
+                                  to disk once the document list changes
+                                  (i.e. new documents are opened, saved or
+                                  closed). This helps to accidentally lose
```

Comment:
I hope it doesn't help to lose stuff, maybe ... to prevent accidentally losing ...

Reply:
Ouch. Thanks for spotting this, I always appreciate your editing pencil!